src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/ActionType.kt

```diff
@@ -42,6 +42,7 @@ data class ActionType<out T>(
 @JvmField val PROJECT_SUMMARY = ActionType("Project Summary") { Unit }
 @JvmField val SEARCH_WITH_JQL = ActionType("Search with JQL") { SearchJqlObservation(it) }
 @JvmField val SEARCH_WITH_JQL_A_STAR = ActionType("Search with JQL a*") { SearchJqlObservation(it) }
+@JvmField val SEARCH_WITH_JQL_WITHOUT_A_START_REPORTER_WAS_ADMIN = ActionType("Search with JQL except a* and reporter was admin") { SearchJqlObservation(it) }
```

Comment:
Why do we call it "reporter was admin"? It can be anyone, not only admin, right?

Reply:
Maybe `SEARCH_WITH_JQL_WITHOUT_A_START_AND_PREVIOUS_REPORTER`?

Reply:
or `SEARCH_WITH_JQL_WITHOUT_WILDCARD_AND_PREVIOUS_REPORTER`